packages/protocol/thread/thread.ts

```diff
@@ -38,6 +38,7 @@ import {
   findAnnotationsResult,
   getHitCountsParameters,
 } from "@replayio/protocol";
+import { features } from "ui/utils/prefs";
```

Comment:
We should avoid adding any imports from `packages/protocol` to things that are outside of `packages/protocol`. Maybe we need a lint rule for this.